NitroxClient/GameLogic/Terrain.cs

```diff
@@ -93,5 +94,19 @@ private IEnumerator WaitAndSyncCells(float delay)
                 yield return new WaitForSeconds(0.05f);
             }
         }
+
+        public IEnumerator WaitForWorldLoad()
```

Comment:
Would mention in a xml comment that this also unfreezes the player as this can't be obtained by its name.